var/spack/repos/builtin/packages/py-kosh/package.py

```diff
@@ -28,3 +30,13 @@ class PyKosh(PythonPackage):
     depends_on("py-llnl-sina@1.11:", type=("build", "run"))
     depends_on("py-networkx", type=("build", "run"))
     depends_on("py-numpy", type=("build", "run"))
+
+    with when("@3:"):
+        depends_on("py-matplotlib")
+        depends_on("py-scipy")
+        depends_on("py-h5py@3:")
+        depends_on("py-scikit-learn@1.0.2:")
+        depends_on("py-pandas")
+        depends_on("py-hdbscan@0.8.29:")
+        depends_on("py-tqdm")
+    patch("version-fix.patch", when="@3.0:3.0.0")  # fixed with v3.0.1
```

Comment:
Do we want to bother with this patch now that 3.0.1 is out?

Reply:
Either we add the patch, or we remove 3.0 from the list of versions. 3.0 doesn't build properly with spack without the patch. Thoughts @doutriaux1 ?  I'm fine just moving to 3.0.1

Reply:
3.0.1 is essentially the same as 3.0, we can skip 3.0.